packages/protocol/scripts/truffle/deploy_release_contracts.ts

```diff
@@ -144,6 +147,92 @@ async function handleGrant(releaseGoldConfig: any, currGrant: number) {
   }
 }
 
+async function checkBalance(releaseGoldConfig: any) {
+  const weiAmountReleasedPerPeriod = new BigNumber(
+    web3.utils.toWei(releaseGoldConfig.amountReleasedPerPeriod.toString())
+  )
+  const grantTotal = weiAmountReleasedPerPeriod
+    .multipliedBy(releaseGoldConfig.numReleasePeriods)
+    .plus(startGold)
+    .toFixed()
+  while (true) {
+    const fromBalance = new BigNumber(await web3.eth.getBalance(fromAddress))
+    if (fromBalance.gte(grantTotal)) {
+      break
+    }
+    console.info(
+      chalk.yellow(
+        fromAddress +
+          "'s balance will not cover the next grant with identifier " +
+          releaseGoldConfig.identifier +
+          '.'
+      )
+    )
+    const addressResponse = await prompts({
+      type: 'text',
+      name: 'newFromAddress',
+      message:
+        'If you would like to continue deployment from a new address please provide it now. If you would like to cancel deployment just press enter. Grants already deployed will persist.',
+    })
+
+    if (addressResponse.newFromAddress === '') {
+      console.info('Exiting because of user input.')
+      process.exit(0)
+    }
+
+    if (!web3.utils.isAddress(addressResponse.newFromAddress)) {
+      console.info(
+        chalk.red('Provided address is invalid, please insert a valid Celo Address.\nRetrying.')
+      )
+      continue
+    }
+    console.info(
+      '\nSending 1 cGLD as a test from ' +
+        fromAddress +
+        ' to ' +
+        addressResponse.newFromAddress +
+        ' to verify ownership.\n'
+    )
+    await web3.eth.sendTransaction({
```

Comment:
Just flagging that we are assuming here that we will never wind up with a leftover balance of < 1 cGLD. Seems like an assumption that is probably safe but may be worth skipping the test transaction if we have < 1 cGLD